test/test_asset_report.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'test_helper'
+
+# Internal: The test for Plaid::AssetReport.
+class PlaidAssetReportTest < PlaidTest
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def test_full_flow
+    # rubocop:enable Metrics/AbcSize, Metrics/MethodLength
```

Comment:
Why do you disable and then enable?

Reply:
I just want to disable the linter for this one method. I can add it as an inline comment instead.